### ISSUE TITLE
Error: Import-NAVContainerLicense with -restart

### ISSUE THREAD

**Issue by git-no** [microsoft/navcontainerhelper#1078]
## Issue
Command `Import-NAVContainerLicense` issues an error if option `-restart` is used.

## Error Message
```powershell
A positional parameter cannot be found that accepts argument 'NAV'.
```

## Reason

Coding failure in file `https://github.com/microsoft/navcontainerhelper/blob/master/ContainerHandling/Import-NavContainerLicense.ps1`, hyphen of Param `ServerInstance` is missing

### Existing Code

```
        if ($restart) {
            Write-Host "Restarting Service Tier"
            Set-NavServerInstance ServerInstance $ServerInstance -restart
        }
```

### Code Correction

```
        if ($restart) {
            Write-Host "Restarting Service Tier"
            Set-NavServerInstance -ServerInstance $ServerInstance -restart
        }
```

## Request

Correction of the code in the file mentioned above would be appreciated.
Many thanks in advance.

**Comment by freddydk** [comment]
Thanks

**Comment by freddydk** [comment]
Shipped in 0.7.0.12

**Comment by pri-kise** [comment]
I think this issue is still present.
I have updated navcontainerhelper to the latest version (0.7.0.23), but I still receive this error:

```
A positional parameter cannot be found that accepts argument 'BC'.
at <ScriptBlock>, <No file>: line 8
A positional parameter cannot be found that accepts argument 'BC'.
In C:\Program Files\WindowsPowerShell\Modules\navcontainerhelper\0.7.0.9\ContainerHandling\Invoke-ScriptInNavContainer.ps1:37 Zeichen:13
+             Invoke-Command -Session $session -ScriptBlock $scriptbloc ...
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Set-NAVServerInstance], ParameterBindingException
    + FullyQualifiedErrorId : PositionalParameterNotFound,Microsoft.Dynamics.Nav.Management.Cmdlets.SetNavServerInstance
```
